src/main.jsx

```diff
@@ -4,17 +4,19 @@ import './styles/main'
 
 import React from 'react'
 import { render } from 'react-dom'
-import { I18n } from './lib/I18n'
 
-import App from './components/App'
+const renderApp = function () {
+  const Root = require('./components/Root').default
+  render(<Root />, document.querySelector('[role=application]'))
+}
 
-const context = window.context
-const lang = document.documentElement.getAttribute('lang') || 'en'
+if (__DEV__) {
+  window.React = React
+  require('preact/devtools')
```

Comment:
It is necessary to be able to use React dev tools.

![image](https://cloud.githubusercontent.com/assets/465582/26099319/a7dd01f4-3a2a-11e7-91cf-1fa063b4eb58.png)

https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi